internal/servers/controller/handlers/hosts/host_service.go

```diff
@@ -456,6 +456,9 @@ func toProto(ctx context.Context, in *static.Host, hostSets []*static.HostSet, o
 			out.HostSetIds = append(out.HostSetIds, hs.GetPublicId())
 		}
 	}
+	if outputFields.Has(globals.HostAddressField) {
```

Comment:
In addition, please add checks to validateUpdateRequest and validateCreateRequest to ensure that the top level address is not passed in.

Reply:
Done!